tests/test_helper/tls.bash

```diff
@@ -0,0 +1,129 @@
+_PKI_DIR="/var/snap/microovn/common/data/pki/"
+CA_CERT_PATH="$_PKI_DIR""cacert.pem"
+CLIENT_CERT_PATH="$_PKI_DIR""client-cert.pem"
+CLIENT_KEY_PATH="$_PKI_DIR""client-privkey.pem"
+CONTROLLER_CERT_PATH="$_PKI_DIR""ovn-controller-cert.pem"
+CONTROLLER_KEY_PATH="$_PKI_DIR""ovn-controller-privkey.pem"
+OVN_NB_CERT_PATH="$_PKI_DIR""ovnnb-cert.pem"
+OVN_NB_KEY_PATH="$_PKI_DIR""ovnnb-privkey.pem"
+OVN_SB_CERT_PATH="$_PKI_DIR""ovnsb-cert.pem"
+OVN_SB_KEY_PATH="$_PKI_DIR""ovnsb-privkey.pem"
+NORTHD_CERT_PATH="$_PKI_DIR""ovn-northd-cert.pem"
+NORTHD_KEY_PATH="$_PKI_DIR""ovn-northd-privkey.pem"
+
+# shellcheck disable=SC2034  # This variable is referenced and passed by name
+declare -g -A OVN_CENTRAL_PKI=(\
+    [$CLIENT_CERT_PATH]=$CLIENT_KEY_PATH\
+    [$CONTROLLER_CERT_PATH]=$CONTROLLER_KEY_PATH\
+    [$OVN_NB_CERT_PATH]=$OVN_NB_KEY_PATH\
+    [$OVN_SB_CERT_PATH]=$OVN_SB_KEY_PATH\
+    [$NORTHD_CERT_PATH]=$NORTHD_KEY_PATH\
+)
+
+# shellcheck disable=SC2034  # This variable is referenced and passed by name
+declare -g -A OVN_CHASSIS_PKI=(\
+     [$CLIENT_CERT_PATH]=$CLIENT_KEY_PATH\
+     [$CONTROLLER_CERT_PATH]=$CONTROLLER_KEY_PATH\
+ )
+
+
+# get_ovn_listening_addr CONTAINER
+#
+# Print an IP address on which OVN services are listening in the specified CONTAINER
+function get_ovn_listening_addr() {
+    local container=$1
+    lxc_exec "$container" "source /var/snap/microovn/common/data/ovn.env && echo \$OVN_LOCAL_IP"
```

Comment:
This will provide you with an address enclosed in brackets (`[]`) when the system is configured for IPv6.

Why not use something like `microovn_get_cluster_address` instead?

Reply:
Thanks for pointing this out. That function was not available initially.

Reply:
Function `get_ovn_listening_addr` removed and usage replaced with `microovn_get_cluster_address`